### ISSUE TITLE
[Feature Request] ARM64 build

### ISSUE THREAD

**Issue by NiQ1** [sysprogs/WinCDEmu#41]
It should be pretty straightforward to compile an ARM64 build since there's no ASM involved but as in the x64 case the kernel driver needs to be signed.

**Comment by xrishox** [comment]
an arm build is seeming more important than ever as arm is becoming more and more relevant.

**Comment by Articulus-Xelmarin** [comment]
Correct, There are more and More arm64 machines, especially with the surface pro being so popular
